src/main/java/uk/gov/register/util/HashValue.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.register.util;
+
+import uk.gov.register.core.HashingAlgorithm;
+
+public class HashValue {
+    private final String rawHash;
+    private final String encodedHash;
+
+    public HashValue(HashingAlgorithm hashingAlgorithm, String rawHash) {
+        this.rawHash = rawHash;
+        this.encodedHash = hashingAlgorithm.toString() + ":" + rawHash;
+    }
```

Comment:
The reason I'm storing the encoded hash is so that it doesn't have to be recomputed on the fly every time it is called.